### ISSUE TITLE
[BUG] Рассинхрон свойств графика и реально указанного в настройках инструмента

### ISSUE THREAD

**Issue by EvgenyMargolin** [alor-broker/Astras-Trading-UI#183]
### Опишите баг

Во вложении 3 последовательных скрина. Задал в настройках LKOH-6.22 в режиме RFUD, сохранил. Открылся график и стакан, но судя по второму скрину - цены не с инструмента LKOH-6.22. Захожу в настройки и вижу SBER (3й скрин).

### Воспроизведение

Создал виджет график. 
Зашел в настройки, указал то что на скрине 1. Сохранил.
Увидел то, что на скрине 2.
Зашел в настройки, увидел то, что на скрине 3.

### Среда.

Браузер - Опера только что обновленная (4й скрин во вложении). ОС - Win7 SP1

### Ожидаемое поведение

При изменении в свойствах на LKOH-6.22 должен меняться и сам график и его заголовок и стакан (при связанных графике и стакане).

### Скриншоты

![1](https://user-images.githubusercontent.com/104624982/169226663-3fca85b9-5f1d-466a-a1f2-96571a3a8a22.png)
![2](https://user-images.githubusercontent.com/104624982/169226723-818b9972-8244-4016-b267-f30df6a8da1d.png)
![3](https://user-images.githubusercontent.com/104624982/169226727-068f168d-4ca4-49db-b8c5-8d9e7d331104.png)

![4](https://user-images.githubusercontent.com/104624982/169228089-3dc08b57-f8d9-4e10-b155-49595adaca6b.png)

### Правила

- [ ] Я согласен следовать правилам

**Comment by PilgrimViis** [comment]
Нет, при изменении настроек графика, стакан менятся точно не должен. Но баг есть, т.к. есть явный рассинхрон между настройками, заголовком виджета и самим графиком. Попробуем разобраться

**Comment by sbelashevskiy** [comment]
**Уточнение**: рассинхрон появляется после открытия настроек виджета
